### DISCUSSION TITLE
is there a plan to add rtl support?

### DISCUSSION THREAD

**Discussion by hosseinjarrahi** [vueform/vueform#91] [Questions] [answered]
is there a plan to add rtl support?
or is there a way to make components  form rtl?

**Comment by adamberecz** [comment, marked as answer]
Vueform inputs are in general support `rtl`. You need to add `dir="rtl"` to `<body>` and all form inputs will adapt. As this is tested less frequently feel free to [open an issue](https://github.com/vueform/vueform/issues/new?assignees=&labels=pending+triage&projects=&template=bug-report.yml) if you find anything that's not the way it supposed to be.
